PowerShell/JumpCloud Module/Docs/Remove-JCUser.md

```diff
@@ -40,6 +50,28 @@ PS C:\> Remove-JCUser cclemons -Force
 ```
 
 Removes the JumpCloud User with Username 'cclemons' using the -Force Parameter. A warning message will not be presented to confirm this operation.
+If the user is a manager, this parameter will clear the manager field for the managed users by this manager.
+
+### Example 3

```

Comment:
We need to note here that this is the same as running 

`Remove-JCUser cclemons -force`